designsystem/src/commonMain/kotlin/org/hisp/dhis/mobile/ui/designsystem/component/InputPhoneNumber.kt

```diff
@@ -0,0 +1,74 @@
+package org.hisp.dhis.mobile.ui.designsystem.component
+
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material3.Icon
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+import org.hisp.dhis.mobile.ui.designsystem.component.internal.RegExValidations
+import org.hisp.dhis.mobile.ui.designsystem.resource.provideDHIS2Icon
+import org.hisp.dhis.mobile.ui.designsystem.resource.provideStringResource
+
+@Composable
+fun InputPhoneNumber(
+    title: String,
+    onCallActionClicked: () -> Unit,
+    modifier: Modifier = Modifier,
+    characterLimit: Int = 10,
+    state: InputShellState = InputShellState.UNFOCUSED,
+    legendData: LegendData? = null,
+    inputText: String? = null,
+    isRequiredField: Boolean = false,
+    onNextClicked: (() -> Unit)? = null,
+    onValueChanged: ((String?) -> Unit)? = null,
+    imeAction: ImeAction = ImeAction.Next,
+    notation: RegExValidations = RegExValidations.ONLY_INTEGERS,
```

Comment:
you will probably need to create a new reg ex  that could be named PHONE_NUMBER that allows for a prefix like +34 or the intended country code,  in the capture app both numbers with country code and without are allowed

Reply:
Yeah, thought about that. I was waiting for Marcos to confirm if it will contain a country code as well along with special characters or just digits. I can make that change once it's confirmed 👍🏾